src/parser/utils.ts

```diff
@@ -27,3 +27,24 @@ export function isMap(d: unknown): d is Map<unknown, unknown> {
 export function isDate(d: unknown): d is Date {
   return Object.prototype.toString.call(d) === '[object Date]';
 }
+
+/** @internal */
+export type StylizeFunction = (x: unknown, style: string) => string;
```

Comment:
```suggestion
export type StylizeFunction = (x: string, style: string) => string;
```

since stylizing takes an existing string already?

Reply:
Currently, getStylizeFunction takes in only a string; however, that is not the only potential case it can be used in the future, e.g. as for non-string types that require custom stylization.

Reply:
Oh I see, I didn't realize these were the Node.js `stylize` function's argument expectations. I'll be sure to have 2 string arguments for my `stylizeFunction` type. I wasn't able to find the documentation on the `stylize` function on the internet could you link me to it?

Reply:
Yeah, `stylize` is definitely underdocumented. It is only mentioned in [the docs for custom inspect() functions in Node.js](https://nodejs.org/api/util.html#custom-inspection-functions-on-objects), but the TS types [do have the right signature](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/cbdc741ac84952a5202b1e822f87ec0017ebd6db/types/node/util.d.ts#L109) and then there’s also always the [actual Node.js source code](https://github.com/nodejs/node/blob/78a15702dd1bc03505ee130b0ad999ee292b6581/lib/internal/util/inspect.js#L560-L572) which doesn’t explicitly say anything but seems to heavily imply to me that the first argument should always be a string (calling it `str` and conditionally either returning that original value or a string that includes it which can only lead to a consistent return type if `str` is always a string to begin with).